dotnet/src/dotnetframework/GxClasses/Core/gxconfig.cs

```diff
@@ -1443,6 +1444,18 @@ internal static bool WrapSingleApiOutput
 					return DefaultWrapSingleApiOutput;
 			}
 		}
+
+		internal static bool SingletonHttpClient()
+		{
+			if (singletonHttpClient == -1)
+			{
+				if (Config.GetValueOrEnvironmentVarOf("SINGLETON_HTTPCLIENT", out string sValue) && int.TryParse(sValue, out int value))
```

Comment:
Maybe "HTTPCLIENT_SINGLETON"? So every HttpClient special config starts with "HTTP_CLIENT_XXX" ?

Reply:
Updated to HTTPCLIENT_SINGLETON for consistency. Thanks!